genomedata/__init__.py

```diff
@@ -494,6 +453,131 @@ def vars(self):
             square(self.means)
 
 
+class _ChromosomeList(object):
+    def __init__(self):
+        self.open_chromosomes = {}  # NB: maintained by subclass
+
+    def __getitem__(self, name):
+        """Returns an existing opened chromosome if it exists"""
+        res = None
+        # If we have opened the chromosome already
+        if (name in self.open_chromosomes and
+           self.open_chromosomes[name].isopen):  # NB: manual closure check
+            # Return cached object
+            res = self.open_chromosomes[name]
```

Comment:
`res` unnecessary in this method, you can just `return` either `self.open_chromosomes[name]` or `None` (`None` is of course unnecessary to return but it adds some clarity